### PR TITLE
Replace icons using svg

### LINKED ISSUE #2068: Replace icons used in geo activities using vectors
#### Software and hardware versions 
Collect v1.13.2 and older versions

#### Problem description
In our geo activities we use png icons so it would be good to remove them and use vectors instead.

Icons I'm talking about:

ic_add_location_black_36dp
ic_my_location_black_36dp
ic_layers_black_36dp
ic_delete_black_36dp
ic_person_pin_circle_black_36dp
ic_place_black_36dp
distance

@opendatakit-bot label "quick win"

### PR DESCRIPTION
**Pull request by nani92**
Closes #2068 

#### What has been done to verify that this works as intended?
Application was tested on my physical device. All SVGs were displayed correctly.
 
#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.

### CODE REVIEW

**Comment by grzesiek2010** [comment]
@nani92 
Good job! Congrats on your first pull request!

I can see only one small problem:
your ic_distance icon is bigger than the original one:

current master branch:
screenshot_2018-04-20-13-50-40

your branch:
screenshot_2018-04-20-13-47-23

I think it should be 36dp like other icons.

File: collect_app/src/main/res/drawable/ic_add_location_black.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="36.000001dp"
+        android:height="36.000001dp"
+        android:viewportWidth="36.000001"
+        android:viewportHeight="36.000001">
```
Comment (grzesiek2010) [inline collect_app/src/main/res/drawable/ic_add_location_black.xml]: Could it be just 36 in 4 lines above? I think that 000001 doesn't change anything.

**Comment by grzesiek2010** [comment]
And another thing... it's my fault because of I didn't list that icon in the description but `ic_pause_black_36dp` is also used in the same places and it's png. Would you like to fix that one also?

**Comment by grzesiek2010** [comment]
It's my fault because I missed it in the description but now I realized there are three other icons used in the same activities:
`ic_pause_black_36dp.png`
`polygon.png`
`polyline.png`

would you like to fix them too? If not we can do that in a separate pull request but it looks as if you are good at it so probably it's not a problem for you :wink:

**Comment by nani92** [comment]
Sorry about wrong size. I'll fix it. 
I'll replace remaining icons as well. ;)

**Comment by grzesiek2010** [comment]
> Sorry about wrong size. I'll fix it.
I'll replace remaining icons as well. ;)

:+1: 

please notice also there are conflicts to solve because we have merged another pull request that changed the same classes.

**Comment by grzesiek2010** [comment]
I tested your changes on two devices. Everything looks ok on Android 5.0.1 but there is a crash on Android 4.2. Please take a look:

> FATAL EXCEPTION: main
                                                                         java.lang.RuntimeException: Unable to start activity ComponentInfo{org.odk.collect.android/org.odk.collect.android.activities.GeoTraceOsmMapActivity}: android.view.InflateException: Binary XML file line #0: Error inflating class <unknown>
                                                                             at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2255)
                                                                             at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2309)
                                                                             at android.app.ActivityThread.access$700(ActivityThread.java:157)
                                                                             at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1289)
                                                                             at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                             at android.os.Looper.loop(Looper.java:176)
                                                                             at android.app.ActivityThread.main(ActivityThread.java:5317)
                                                                             at java.lang.reflect.Method.invokeNative(Native Method)
                                                                             at java.lang.reflect.Method.invoke(Method.java:511)
                                                                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1102)
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:869)
                                                                             at dalvik.system.NativeStart.main(Native Method)
                                                                          Caused by: android.view.InflateException: Binary XML file line #0: Error inflating class <unknown>
                                                                             at android.view.LayoutInflater.createView(LayoutInflater.java:619)
                                                                             at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:56)
                                                                             at android.view.LayoutInflater.onCreateView(LayoutInflater.java:666)
                                                                             at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:691)
                                                                             at android.view.LayoutInflater.rInflate(LayoutInflater.java:752)
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:495)
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:353)
                                                                             at com.android.internal.policy.impl.PhoneWindow.setContentView(PhoneWindow.java:360)
                                                                             at android.app.Activity.setContentView(Activity.java:1932)
                                                                             at org.odk.collect.android.activities.GeoTraceOsmMapActivity.onCreate(GeoTraceOsmMapActivity.java:109)
                                                                             at android.app.Activity.performCreate(Activity.java:5326)
                                                                             at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1097)
                                                                             at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2218)
                                                                             at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2309) 
                                                                             at android.app.ActivityThread.access$700(ActivityThread.java:157) 
                                                                             at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1289) 
                                                                             at android.os.Handler.dispatchMessage(Handler.java:99) 
                                                                             at android.os.Looper.loop(Looper.java:176) 
                                                                             at android.app.ActivityThread.main(ActivityThread.java:5317) 
                                                                             at java.lang.reflect.Method.invokeNative(Native Method) 
                                                                             at java.lang.reflect.Method.invoke(Method.java:511) 
                                                                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1102) 
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:869) 
                                                                             at dalvik.system.NativeStart.main(Native Method) 
                                                                          Caused by: java.lang.reflect.InvocationTargetException
                                                                             at java.lang.reflect.Constructor.constructNative(Native Method)
                                                                             at java.lang.reflect.Constructor.newInstance(Constructor.java:417)
                                                                             at android.view.LayoutInflater.createView(LayoutInflater.java:593)
                                                                             at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:56) 
                                                                             at android.view.LayoutInflater.onCreateView(LayoutInflater.java:666) 
                                                                             at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:691) 
                                                                             at android.view.LayoutInflater.rInflate(LayoutInflater.java:752) 
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:495) 
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:397) 
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:353) 
                                                                             at com.android.internal.policy.impl.PhoneWindow.setContentView(PhoneWindow.java:360) 
                                                                             at android.app.Activity.setContentView(Activity.java:1932) 
                                                                             at org.odk.collect.android.activities.GeoTraceOsmMapActivity.onCreate(GeoTraceOsmMapActivity.java:109) 
                                                                             at android.app.Activity.performCreate(Activity.java:5326) 
                                                                             at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1097) 
                                                                             at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2218) 
                                                                             at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2309) 
                                                                             at android.app.ActivityThread.access$700(ActivityThread.java:157) 
                                                                             at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1289) 
                                                                             at android.os.Handler.dispatchMessage(Handler.java:99) 
                                                                             at android.os.Looper.loop(Looper.java:176) 
                                                                             at android.app.ActivityThread.main(ActivityThread.java:5317) 
                                                                             at java.lang.reflect.Method.invokeNative(Native Method) 
                                                                             at java.lang.reflect.Method.invoke(Method.java:511) 
                                                                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1102) 
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:869) 
                                                                             at dalvik.system.NativeStart.main(Native Method) 
                                                                          Caused by: android.content.res.Resources$NotFoundException: File res/drawable/ic_distance.xml from drawable resource ID #0x7f08008e
                                                                             at android.content.res.Resources.loadDrawable(Resources.java:2842)
                                                                             at android.content.res.TypedArray.getDrawable(TypedArray.java:602)
                                                                             at android.widget.TextView.<init>(TextView.java:1027)
                                                                             at android.widget.Button.<init>(Button.java:126)
                                                                             at android.widget.Button.<init>(Button.java:122)
                                                                             at java.lang.reflect.Constructor.constructNative(Native Method) 
                                                                             at java.lang.reflect.Constructor.newInstance(Constructor.java:417) 
                                                                             at android.view.LayoutInflater.createView(LayoutInflater.java:593) 
                                                                             at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:56) 
                                                                             at android.view.LayoutInflater.onCreateView(LayoutInflater.java:666) 
                                                                             at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:691) 
                                                                             at android.view.LayoutInflater.rInflate(LayoutInflater.java:752) 
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:495) 
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:397) 
                                                                             at android.view.LayoutInflater.inflate(LayoutInflater.java:353) 
                                                                             at com.android.internal.policy.impl.PhoneWindow.setContentView(PhoneWindow.java:360) 
                                                                             at android.app.Activity.setContentView(Activity.java:1932) 
                                                                             at org.odk.collect.android.activities.GeoTraceOsmMapActivity.onCreate(GeoTraceOsmMapActivity.java:109) 
                                                                             at android.app.Activity.performCreate(Activity.java:5326) 
                                                                             at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1097) 
                                                                             at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2218) 
                                                                             at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2309) 
                                                                             at android.app.ActivityThread.access$700(ActivityThread.java:157) 
                                                                             at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1289) 
                                                                             at android.os.Handler.dispatchMessage(Handler.java:99) 
                                                                             at android.os.Looper.loop(Looper.java:176) 
                                                                             at android.app.ActivityThread.main(ActivityThread.java:5317) 
                                                                             at java.lang.reflect.Method.invokeNative(Native Method) 
                                                                             at java.lang.reflect.Method.invoke(Method.java:511) 
                                                                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1102) 
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:869) 
                                                                             at dalvik.system.NativeStart.main(Native Method) 
                                                                          Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #0: invalid drawable tag vector
                                                                             at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:917)
                                                                             at android.graphics.drawable.Drawable.createFromXml(Drawable.java:858)
                                                                             at android.content.res.Resources.loadDrawable(Resources.java:2839)

**Comment by nani92** [comment]
Hi,
I don't have jelly device and because of known issue I'm not able to run  emulator. See here: https://github.com/opendatakit/collect/issues/706 

The lowest API i could use on emulator is kitkat. Error occurred there as well and now is fixed.

**Comment by grzesiek2010** [comment]
@nani92 
unfortunately, the same crash still occurs.

**Review by grzesiek2010 — CHANGES_REQUESTED** [review]
@nani92
unfortunately, the same crash still occurs.

**Comment by nani92** [comment]
My bad :( Sorry for that.
Should be fine now.

**Comment by huangyz0918** [comment]
@nani92 I guess you faced a random CI problem, I think you can add an extra commit to redo the CI and pass the check 😄

**Comment by grzesiek2010** [comment]
> I think you can add an extra commit to redo the CI and pass the check 

you can just click `Rerun job` button.

**Comment by lakshyagupta21** [comment]
> you can just click Rerun job button

@grzesiek2010 I'm not sure if @nani92 has the privilege to rerun the job, only admins have access to that button.

@nani92 You can still rerun the CI without adding any extra commit by changing the latest commit name and followed by force push.
You can find more at [this link](https://help.github.com/articles/changing-a-commit-message/)

**Comment by nani92** [comment]
@grzesiek2010  if you talking about "Rerun job with SHH" on CircleCI page it's disabled for me and says I need write permissions.

**Comment by grzesiek2010** [comment]
Thanks! now it looks ok. But it would be good to add more info about the problem we ran into. I mean you can edit your commit message describing which Android versions need that wrappers or you can add comments just in files. I think it's needed because it will allow us to remove those wrappers one we stop supporting affected Android versions.

**Comment by grzesiek2010** [comment]
> if you talking about "Rerun job with SHH" on CircleCI page it's disabled for me and says I need write permissions.

you must be logged in using the same Github account (I mean in CircleCI).

**Comment by nani92** [comment]
Thanks @lakshyagupta21 i'll try it right away. :)

**Comment by grzesiek2010** [comment]
> I'm not sure if @nani92 has the privilege to rerun the job, only admins have access to that button.

I'm not an admin and I have like lots of other contributors. Everyone can rerun their own job on their own branch.

**Comment by nani92** [comment]
> Thanks! now it looks ok. But it would be good to add more info about the problem we ran into. I mean you can edit your commit message describing which Android versions need that wrappers or you can add comments just in files. I think it's needed because it will allow us to remove those wrappers one we stop supporting affected Android versions.

Sure @grzesiek2010 ! I will do it :)

**Review by grzesiek2010 — APPROVED** [review]

**Comment by kkrawczyk123** [comment]
###Tested with success!

Verified on Androids: 4.1, 4.2, 4.4, 5.1, 6.0, 7.0 and 8.1.

@opendatakit-bot unlabel "needs testing"
@opendatakit-bot label "behavior verified"

**Review by lognaturel — APPROVED** [review]

**Comment by lognaturel** [comment]
Thank you very much @nani92! If you haven't already, consider joining the developer Slack at http://slack.opendatakit.org/ and checking out the forum at https://forum.opendatakit.org/.

**Pull request merged**